test/e2e/windows/memory_limits.go

```diff
@@ -104,32 +104,38 @@ func overrideAllocatableMemoryTest(f *framework.Framework, allocatablePods int)
 	const (
 		podType = "memory_limit_test_pod"
 	)
-
-	totalAllocatable := getTotalAllocatableMemory(f)
-
-	memValue := totalAllocatable.Value()
-	memPerPod := memValue / int64(allocatablePods)
-	ginkgo.By(fmt.Sprintf("Deploying %d pods with mem limit %v, then one additional pod", allocatablePods, memPerPod))
-
-	// these should all work
-	pods := newMemLimitTestPods(allocatablePods, imageutils.GetPauseImageName(), podType, strconv.FormatInt(memPerPod, 10))
-	f.PodClient().CreateBatch(pods)
-
-	failurePods := newMemLimitTestPods(1, imageutils.GetPauseImageName(), podType, strconv.FormatInt(memPerPod, 10))
-	f.PodClient().Create(failurePods[0])
-
-	gomega.Eventually(func() bool {
-		eventList, err := f.ClientSet.CoreV1().Events(f.Namespace.Name).List(context.TODO(), metav1.ListOptions{})
-		framework.ExpectNoError(err)
-		for _, e := range eventList.Items {
-			// Look for an event that shows FailedScheduling
-			if e.Type == "Warning" && e.Reason == "FailedScheduling" && e.InvolvedObject.Name == failurePods[0].ObjectMeta.Name {
-				framework.Logf("Found %+v event with message %+v", e.Reason, e.Message)
-				return true
+	//
+	selector := labels.Set{"kubernetes.io/os": "windows"}.AsSelector()
+	nodeList, err := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{
+		LabelSelector: selector.String(),
+	})
+	framework.ExpectNoError(err)
+	totalAllocatable := resource.NewQuantity(0, resource.BinarySI)
```

Comment:
This isn't required as will "fill up" memory for each node by deploying a single pod that uses the memory and don't need to calculate the total.

Reply:
@jsturtevant could you please elaborate on this point, I am unable to understand your point here.

Reply:
This isn't used because we are doing it per node now, total of all nodes. We could add it up just and keep it for logging but 
the spot below should be simplified to:

`nodeAllocatable := status.Allocatable[v1.ResourceMemory]`